### DISCUSSION TITLE
upload images with tip tap - Vuetify/vue.js it is not CLI version and laravel.

### DISCUSSION THREAD

**Discussion by Hantmaes** [ueberdosis/tiptap#835] [General Questions]
Could you please help me? I am trying to upload images with tip tap, but on the official website there is no information about it, the way it is available is via the url. My implementation is using Vuetify/vue.js it is not CLI version and laravel.

Thank you very much for your attention.

**Comment by hanspagel** [comment]
Here is an example:
https://gist.github.com/slava-vishnyakov/16076dff1a77ddaca93c4bccd4ec4521

Hope that helps!
